textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.common.constants;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.List;
+
+import edu.uci.ics.textdb.api.common.IField;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.common.field.Attribute;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.DateField;
+import edu.uci.ics.textdb.common.field.DoubleField;
+import edu.uci.ics.textdb.common.field.FieldType;
+import edu.uci.ics.textdb.common.field.IntegerField;
+import edu.uci.ics.textdb.common.field.StringField;
+
+/**
+ * @author sandeepreddy602
+ * Including this class in src/main/java since it is required by other projects
+ * Keeping it in src/test/java doesn't make it available to the other projects
+ */
+public class TestConstants {
+    // Sample Fields
+    public static final String FIRST_NAME = "firstName";
+    public static final String LAST_NAME = "lastName";
+    public static final String AGE = "age";
+    public static final String HEIGHT = "height";
+    public static final String DATE_OF_BIRTH = "dateOfBirth";
+    
+    
+    public static final Attribute FIRST_NAME_ATTR = new Attribute(FIRST_NAME, FieldType.STRING);
+    public static final Attribute LAST_NAME_ATTR = new Attribute(LAST_NAME, FieldType.STRING);
+    public static final Attribute AGE_ATTR = new Attribute(AGE, FieldType.INTEGER);
+    public static final Attribute HEIGHT_ATTR = new Attribute(HEIGHT, FieldType.DOUBLE);
+    public static final Attribute DATE_OF_BIRTH_ATTR = new Attribute(DATE_OF_BIRTH, FieldType.DATE);
+
+    // Sample Schema
+    public static final List<Attribute> SAMPLE_SCHEMA = Arrays.asList(

```

Comment:
Can we rename "SAMPLE_SCHEMA" to "SAMPLE_SCHEMA_PEOPLE" in case we may introduce other schemas?